### DISCUSSION TITLE
大佬可以支持一下中文的演员名刮削吗, 目前只支持日文

### DISCUSSION THREAD

**Discussion by xingfanxia** [ddd354/JAVnfoMoviesImporter.bundle#7] [Ideas]
大佬可以支持一下中文的演员名刮削吗, 目前只支持日文
![Plex 2022-04-12 21-30-04](https://user-images.githubusercontent.com/8936233/163101600-bafa8dab-b5b4-47bf-b125-6d366cf73fb2.jpg)

**Comment by xingfanxia** [comment]
看了下源码和`filetree.json` `filetree.json` 已经包括了简繁中文的女优名字了
在 `__init__.py` 的map查找不应该没有照片..大佬可以看下这是什么bug吗

**Comment by xingfanxia** [comment]
@ddd354 ^^多谢

**Comment by xingfanxia** [comment]
<img width="258" alt="image" src="https://user-images.githubusercontent.com/8936233/163381997-b3ad5956-e5ea-4ce4-af83-ae62715294e2.png">

![Filetree json 2022-04-14 04-28-26](https://user-images.githubusercontent.com/8936233/163382070-9a2b3a2c-57ab-465b-8b04-4dcad1bf3a41.jpg)

**Comment by ddd354** [comment]
本地测试没问题的 下载最新版&网络问题也会导致刮不出来

**Comment by xingfanxia** [comment]
@ddd354 ok 有什么办法retry吗
